doc/rtd/development/integration_tests.rst

```diff
@@ -31,26 +31,179 @@ integration tests. To run all integration tests, you would run:
 
 .. code-block:: bash
 
-    $ tox -e integration-tests
+    tox -e integration-tests
 
-``pytest`` arguments may also be passed. For example:
+``pytest`` arguments may also be passed.
+
+Or, specify a path to run all tests inside the given file or directory:
```

Comment:
I'm not sure we need 3 separate examples here showing how to run a test. One of the benefits of using pytest and tox is that things "just work" like they do for any other pytest or tox project. I think one example here would suffice with perhaps a link to https://docs.pytest.org/en/7.1.x/how-to/usage.html

Reply:
Hmm... I might be swayed by this, but still think it is worth directly calling out, because I think specifying a path is one of the most common use cases for integration tests. Rarely will a new contributor be running the entire integrations sweet due to the insanely long run time? 

Let me see how this looks in tabular form so it takes up less space and maybe this will feel less like a waste of space.

Reply:
So worst comes to worst, I agree, 3 different examples is not necessary and just one command showing how to specify a path would suffice.